lhotse/recipes/ami.py

```diff
@@ -207,9 +207,14 @@ def download_audio(
                     wav_dir = target_dir / "wav_db" / item / "audio"
                     wav_dir.mkdir(parents=True, exist_ok=True)
                     wav_path = wav_dir / wav_name
-                    resumable_download(
-                        wav_url, filename=wav_path, force_download=force_download
-                    )
+                    try:
+                        resumable_download(
+                            wav_url, filename=wav_path, force_download=force_download
+                        )
+                    except:
```

Comment:
PEP8 does not allow a bare `except` clause (see [this](https://stackoverflow.com/questions/14797375/should-i-always-specify-an-exception-type-in-except-statements/14797508#14797508) for why you should specify an exception type).

Reply:
ok thanks! the next commit should fix the issue

Reply:
Thanks. Is this ready to merge?